src/main/java/org/hawkular/client/android/fragment/MetricAvailabilityFragment.java

```diff
@@ -172,20 +252,28 @@ private void setUpChartColumns() {
         List<AxisValue> chartAxisValues = getChartAxisValues();
 
         ColumnChartData chartData = new ColumnChartData()
-            .setColumns(chartColumns);
+                .setColumns(chartColumns);
         chartData.setAxisXBottom(new Axis()
-            .setValues(chartAxisPoints));
+                .setValues(chartAxisPoints));
         chartData.setAxisYLeft(new Axis()
-            .setValues(chartAxisValues));
+                .setValues(chartAxisValues));
 
         chart.setColumnChartData(chartData);
     }
 
     private List<Column> getChartColumns() {
-        List<Column> chartColumns = new ArrayList<>(metricData.size());
+        List<Column> chartColumns = new ArrayList<>(metricBucket.size());
+
+        for (MetricAvailabilityBucket metricBucket : this.metricBucket) {
+            MetricAvailability metricAvailability = null;
+            if (metricBucket.getValue().equals("NaN")) {
+                metricAvailability = MetricAvailability.from("unknown");
+            } else if (Float.parseFloat(metricBucket.getValue()) >= .5) {
```

Comment:
There is now since hawkular-services 0.0.5 also a value "admin" available.
You should probably test this whole section against 0.0.5